src/features/landing/components/AuthSection/AuthFormComponent/AuthFormComponent.tsx

```
@@ -0,0 +1,44 @@
+import { CreateUserPayload, SignUpVariables } from './types';
+
+function AuthFormComponent(): React.ReactElement {
+  const [isAuthenticated, setIsAuthenticated] = React.useState(false);
+  const [signupMutation, { loading }] = useMutation<CreateUserPayload, SignUpVariables>(
+    SIGNUP_MUTATION
+  );
+
+  return (
+    <Box sx={styles.formWrapper}>
+      {loading && (
+        <Box sx={styles.loader} role="status">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)